### PR TITLE
Hide Tag Parents for non-topic tags

### PR DESCRIPTION
**Pull request by clloyd**

### CODE REVIEW

**Comment by clloyd** [comment]
fixes #116

File: public/components/TagContext/TagRelationship/TagRelationship.react.js [hunk 1]
```diff
@@ -100,6 +101,10 @@ export default class TagRelationship extends React.Component {
         return false;
       }
 
+      if (this.props.tag.type !== tagTypes.topic.name) {
```
Comment (currysoup) [inline public/components/TagContext/TagRelationship/TagRelationship.react.js]: I've seen topics being mapped to keyword tags. Does this mean this rule should also compare to keyword tags?

Comment (clloyd) [inline public/components/TagContext/TagRelationship/TagRelationship.react.js]: No, in this case it's fine, they're only being mapped to Keywords when they're crossing the boundaries to old systems.

**Comment by currysoup** [comment]
:+1:

**Pull request merged**